src/test/ui/unboxed-closures/unboxed-closure-sugar-region.stderr

```diff
@@ -1,8 +1,11 @@
-error[E0107]: wrong number of lifetime arguments: expected 1, found 0
+error[E0107]: wrong number of lifetime arguments for `Foo`: expected 1, found 0
   --> $DIR/unboxed-closure-sugar-region.rs:30:51
    |
+LL | trait Foo<'a,T> {
+   | --------------- defined here, with 1 lifetime parameter: `'a`
+...
 LL | fn test2(x: &dyn Foo<(isize,),Output=()>, y: &dyn Foo(isize)) {
-   |                                                   ^^^^^^^^^^ expected 1 lifetime argument
+   |                                                   ^^^^^^^^^^ found no lifetime argument
```

Comment:
I think the previous wording was slightly easier to understand (but the span was bad).

We probably won't deal with this now, but it'd be ideal if we could suggest the a new lifetime in the right place:

```
fn test2<'a>(x: &'a dyn Foo<(isize,),Output=()>, y: &'a dyn Foo<'a, isize>) {
```

Reply:
improved 🎉